src/app/authorization/login/login.component.ts

```diff
@@ -46,8 +47,8 @@ export class LoginComponent implements OnInit, OnDestroy {
 
   ngOnInit(): void {
     this.createForm();
-    if (localStorage.getItem('registeruser')?.length) {
-      let data: any = localStorage.getItem('registeruser');
+    if (localStorage.getItem('registerUser')?.length) {
+      let data: any = localStorage.getItem('registerUser');
```

Comment:
can we write condition like 
this.userData = JSON.parse(localstore.getitem('registerUser') as string ) ?